.github/workflows/vast.yaml

```diff
@@ -164,10 +164,29 @@ jobs:
         uses: abatilo/actions-poetry@v2.1.6
         with:
           poetry-version: 1.1.14
+      - name: Setup R
+        uses: r-lib/actions/setup-r@v2
+      - name: Setup R dependencies
+        uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          working-directory: web
+          cache-version: 2
+      # The setup-r-dependencies action leaves lingering state in web/.github.
+      # Turns out we clone our tenzir/.github repo into there during the website
+      # build, so must clean up prior to doing so. The Right Fix is a one-line
+      # PR against r-lib/actions to remove the .github dir at action exit.
+      - name: Clean up after setup-r-dependencies
+        run: rm -rf web/.github
+      - name: Setup Quarto
+        uses: quarto-dev/quarto-actions/setup@v2
+      - name: Render Quarto Project
+        uses: quarto-dev/quarto-actions/render@v2
+        with:
+          path: web
```

Comment:
This shouldn't be needed anymore, it's already included in `make`, which `yarn start` invokes.